wagtail_transfer/auth.py

```diff
@@ -31,3 +31,9 @@ def digest_for_source(source, message):
         message = message.encode()
 
     return hmac.new(key, message, hashlib.sha1).hexdigest()
+
+def requests_auth(source):
```

Comment:
This is a pretty broad `try/except` for just retrieving a value or `None`. Could you catch eg `KeyError` instead?

Reply:
I guess so... presumably any other errors, if present, will pop up in the settings or requests modules.

Another option, if you prefer it, is to use dict.get() with a default:

`  return settings.WAGTAILTRANSFER_SOURCES[source].get('BASIC_AUTH', None)`

Reply:
Hi Jacob, it's been a busy week... finally made the above change.